### ISSUE TITLE
Chrome moving to MV3 some time this year

### ISSUE THREAD

**Issue by Mehtuhl** [AlyoshaVasilieva/luminous-ttv-ext#5]
Is the extension and server gonna function once chrome goes to MV3? Error log says it's gonna be deprecated.
If not, are we gonna have to use it on firefox or will there be some fixes?

**Comment by AlyoshaVasilieva** [comment]
I didn't expect this project to last so long. I'm pretty sure I can rewrite the extension to MV3, but I think I have to drop retries/fallback to ads, so if the server is down the extension will break Twitch.

**Comment by Mehtuhl** [comment]
Yeah your program has been the most reliable proxy adblock I've ever used. All the others fail at some point. Kinda sucks that twitch is really adamant on forcing people to watch ads.

Whatever you do is much appreciated. Firefox is always an alternative either way.
